Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshCellInterfaceTest.cxx

```diff
@@ -66,7 +66,7 @@ class CustomQEPolyVisitor
 // Test the cell interface
 template <typename TCell>
 int
-TestCellInterface(std::string name, TCell * aCell)
+TestCellInterface(const std::string & name, TCell * aCell)
```

Comment:
...and the `name` parameter of _this_ TestCellInterface may also be `string_view`. (I guess it should also be placed in a different namespace, to avoid name conflicts with the _other_ TestCellInterface function template, but apparently it "just" worked. That's probably beyond the scope of this PR anyway.)

Reply:
Done.